src/dms-preview/azext_dms/_help.py

```diff
@@ -138,6 +201,32 @@
                 "databaseName": "database name", // if this is missing, it will default to the 'postgres' database
                 "port": 5432                // if this is missing, it will default to 5432
             }
+        - name: The format of the connection JSON object for MongoDB connections.
+          text: >
+            {
+                "userName": null,   // if this is missing or null, you will be prompted
+                "password": null,   // if this is missing or null (highly recommended) you will be prompted
+                "connectionString": "mongodb://hostOrIp:port"
+            }
+"""
+
+helps['dms project task cancel'] = """
+    type: command
+    short-summary: This command is being deprecated. Use the stop command instead.
```

Comment:
Isn't this command being added with this PR? If it's deprecated, why include it?

Reply:
The "cancel" command exists in the main CLI module. We've had a change of nomenclature to bring a more unified experience among UI, PowerShell, and CLI. I thought until this set of commands goes GA (i.e. copied out of the extension and into the main module) I would offer this message for anyone who might see it.

Reply:
I see, in that case, please remember that when these commands go GA and into the CLI module, you will still need to deprecate the commands from the CLI side for some time to not break users who have not used the extension.